tests/deepspeed/test_deepspeed.py

```diff
@@ -1008,14 +1009,14 @@ def test_load_best_model(self, stage):
             """.split()
         args.extend(["--source_prefix", "translate English to Romanian: "])
 
-        ds_args = f"--deepspeed {self.test_file_dir_str}/ds_config_zero3.json".split()
+        ds_args = f"--deepspeed {self.test_file_dir_str}/ds_config_{stage}.json".split()
         script = [f"{self.examples_dir_str}/pytorch/translation/run_translation.py"]
         launcher = get_launcher(distributed=False)
 
         cmd = launcher + script + args + ds_args
         # keep for quick debug
         # print(" ".join([f"\nPYTHONPATH={self.src_dir_str}"] +cmd)); die
-        with CaptureStderr() as cs:
+        with CaptureStd() as cs:
             execute_subprocess_async(cmd, env=self.get_env())
         # enough to test it didn't fail
-        assert "Detected DeepSpeed ZeRO-3" in cs.err
+        assert "DeepSpeed info:" in cs.out
```

Comment:
Let's move to a `self.assertIn` since we are using a unittest.

Reply:
Good catch. Thank you, Sylvain!